pkg/hive/cell/structured.go

```diff
@@ -568,15 +568,15 @@ type subReporter struct {
 
 func (s *subReporter) OK(message string) {
 	if err := s.base.setStatus(s.id, StatusOK, message, nil); err != nil {
-		log.WithError(err).Warnf("could not set OK status on subreporter %q", s.id)
+		log.WithError(err).Debugf("could not set OK status on subreporter %q", s.id)
```

Comment:
Why not check the error returned is indeed because the reporter is stopped, rather than blindly treating all errors as the same? i.e. https://github.com/cilium/cilium/issues/31238

Reply:
@christarazi Makes sense, made the change, ptal.